library/system.lisp

```diff
@@ -74,57 +61,86 @@ While the result will always contain microseconds, some implementations may retu
 
 (coalton-toplevel
 
-  (declare get-run-time (Unit -> UFix))
+  (declare get-run-time (Unit -> Integer))
   (define (get-run-time)
-    "Gets the run-time."
-    (lisp UFix ()
+    "Gets the run-time in `internal time units`. This is implementation specific: it may measure real time, run time, CPU cycles, or some other quantity. The difference between two successive calls to this function represents quantity accumulated during that period of time."
+    (lisp Integer ()
       (cl:get-internal-run-time)))
 
-  (declare get-real-time (Unit -> UFix))
+  (declare get-real-time (Unit -> Integer))
   (define (get-real-time)
-    "Gets the real-time."
-    (lisp UFix ()
+    "Gets the real-time in `internal time units`. The difference between two successive calls to this function represents the time that has elapsed."
+    (lisp Integer ()
       (cl:get-internal-real-time)))
 
-  #+sbcl
-  (declare get-bytes-consed (Unit -> UFix))
-  #+sbcl
-  (define (get-bytes-consed)
-    "Gets the number of bytes consed (only implemented for SBCL"
-    (lisp UFix ()
-      (sb-ext:get-bytes-consed)))
+  (declare internal-time-units-per-second (Unit -> Integer))
+  (define (internal-time-units-per-second)
```

Comment:
Maybe this should just be a variable and not a function, since its value is constant.